### PR TITLE
drop labels for which no glyphs are available

### PR DESCRIPTION
**Pull request by hjanetzek**

### CODE REVIEW

File: core/src/text/fontContext.cpp [hunk 1]
```diff
@@ -185,6 +185,11 @@ bool FontContext::layoutText(TextStyle::Parameters& _params, const std::string&
                      (metrics.aabb.y + height * 0.5) * TextVertex::position_scale);
 
     auto it = _quads.begin() + quadsStart;
+    if (it == _quads.end()) {
```
Comment (tallytalwar) [inline core/src/text/fontContext.cpp]: Why not just check for size of `_quads` at the beginning before doing any textWrapping stuff?

Comment (hjanetzek) [inline core/src/text/fontContext.cpp]: The _quads are set during textWrap/drawShapeRange. 

Actually the line.shapes().size() test at the beginning already checks for no-glyphs-at-all. So this new test only catches lines for which only whitespaces were found (and this really happend when no fallbacks were loaded, making the map unusable slow for Cairo). 

Should we rather drop labels if they have any missing glyphs?

Comment (tallytalwar) [inline core/src/text/fontContext.cpp]: What will happen currently if a label is missing a glyph, will it show a default quad or an empty space?

If so, then yes it makes sense to drop the label if its missing any glyph required by it to be rendered completely.

Comment (hjanetzek) [inline core/src/text/fontContext.cpp]: Missing glyphs will just be ignored - i.e. not added to the LineLayout and not adding 'empty space' to it

**Pull request merged**